pyproject.toml

```diff
@@ -27,8 +27,8 @@ requires-python = ">=3.6,<3.11"
 requires = [
   "dataclasses==0.8; python_version == '3.6'",
   "filelock>=3.0",
-  "ipython>=5.5.0,<8.0",
-  "ipywidgets>=7.0.0,<8.0",
+  "ipython>=5.5.0,<9.0",
```

Comment:
I'll give this a spin, but a major version bump could mean disaster. Have you reviewed the changelogs? Especially of ipywidgets? It's generally good form to have done so and reported that in either the change description or a PR comment. Right now we just hope we're getting lucky and the existing tests catch any API breaks the upgrade triggers.

Reply:
Makes sense - wasn't sure about the test coverage/how best to smoke-test. I'll take a look at the changelogs for `ipywidgets` and circle back.